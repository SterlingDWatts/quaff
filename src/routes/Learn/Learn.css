.Learn {
  height: 100%;
}

.Learn--info-text {
  grid-column-end: span 4;
  line-height: 1;
  color: grey;
  font-family: "Lekton", sans-serif;
}

.Learn .TopicSquare__top_label,
.Learn .TopicSquare__container {
  color: white;
  background-color: #720d5d;
}

.Learn .TopicSquare__container--locked {
  color: grey;
  background-color: white;
}

@media screen and (min-width: 500px) {
  .Learn--info-text {
    grid-column-end: span 6;
  }
}

@media screen and (min-width: 1000px) {
  .Learn--info-text {
    grid-column-end: span 8;
  }
}
