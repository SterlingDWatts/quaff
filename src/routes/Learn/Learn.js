import React, { Component } from "react";
import SubNavBar from "../../components/SubNavBar/SubNavBar";
import { ExploreSquare, TopicSquare } from "../../components/Utils/Utils";
import "./Learn.css";

class Learn extends Component {
  render() {
    return (
      <div className="Learn">
        <SubNavBar />
        <ExploreSquare>
          <TopicSquare
            to="/learn/viticulture"
            label="Viticulture"
            picture="https://sterlingdwatts.github.io/quaff_quizz/images/viticulture2.jpg"
          />
          <TopicSquare
            to="/learn/winemaking"
            label="Winemaking"
            picture="https://sterlingdwatts.github.io/quaff_quizz/images/winemaking2.jpg"
          />
          <TopicSquare
            to="/learn/california"
            label="California"
            picture="https://sterlingdwatts.github.io/quaff_quizz/images/california2.jpg"
          />
          <TopicSquare
            to="/learn/bordeaux"
            label="Bordeaux"
            picture="https://sterlingdwatts.github.io/quaff_quizz/images/france2.jpg"
          />
          <TopicSquare
            to="/learn/viticulture"
            label="Viticulture"
            picture="https://sterlingdwatts.github.io/quaff_quizz/images/viticulture2.jpg"
          />
          <TopicSquare
            to="/learn/winemaking"
            label="Winemaking"
            picture="https://sterlingdwatts.github.io/quaff_quizz/images/winemaking2.jpg"
          />
        </ExploreSquare>
      </div>
    );
  }
}

export default Learn;
