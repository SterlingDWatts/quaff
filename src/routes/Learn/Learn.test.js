import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import Learn from "./Learn";

describe("Learn Component", () => {
  it("render without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Learn />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
