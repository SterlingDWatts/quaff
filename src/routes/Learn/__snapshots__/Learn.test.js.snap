// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Learn Component render the UI as expected 1`] = `
<div
  className="Learn"
>
  <div
    className="ExploreSquare "
  >
    <div
      className="ExploreSquare__border"
    >
      <div
        className="ExploreSquare__container"
      >
        <div
          className="ExploreSquare__grid"
        >
          <div
            className="Learn--info-text"
          >
            <a
              href="/login"
              onClick={[Function]}
            >
              Login
            </a>
             to unlock additional content.
             
            <a
              href="/create-account"
              onClick={[Function]}
            >
              Create an account
            </a>
             if you don't have one.
          </div>
          <a
            className="TopicSquare "
            href="/learn"
            onClick={[Function]}
          >
            <div
              className="TopicSquare__picture_container"
            >
              <div
                className="TopicSquare__picture TopicSquare__picture--locked"
                style={
                  Object {
                    "backgroundImage": "url(https://i.imgur.com/h6jaH4c.jpg)",
                  }
                }
              />
              <div
                className="TopicSquare__container TopicSquare__container--locked"
              >
                Coming soon
              </div>
            </div>
          </a>
          <a
            className="TopicSquare "
            href="/learn"
            onClick={[Function]}
          >
            <div
              className="TopicSquare__picture_container"
            >
              <div
                className="TopicSquare__picture TopicSquare__picture--locked"
                style={
                  Object {
                    "backgroundImage": "url(https://i.imgur.com/h6jaH4c.jpg)",
                  }
                }
              />
              <div
                className="TopicSquare__container TopicSquare__container--locked"
              >
                Coming soon
              </div>
            </div>
          </a>
          <a
            className="TopicSquare "
            href="/learn"
            onClick={[Function]}
          >
            <div
              className="TopicSquare__picture_container"
            >
              <div
                className="TopicSquare__picture TopicSquare__picture--locked"
                style={
                  Object {
                    "backgroundImage": "url(https://i.imgur.com/h6jaH4c.jpg)",
                  }
                }
              />
              <div
                className="TopicSquare__container TopicSquare__container--locked"
              >
                Coming soon
              </div>
            </div>
          </a>
          <a
            className="TopicSquare "
            href="/learn"
            onClick={[Function]}
          >
            <div
              className="TopicSquare__picture_container"
            >
              <div
                className="TopicSquare__picture TopicSquare__picture--locked"
                style={
                  Object {
                    "backgroundImage": "url(https://i.imgur.com/h6jaH4c.jpg)",
                  }
                }
              />
              <div
                className="TopicSquare__container TopicSquare__container--locked"
              >
                Coming soon
              </div>
            </div>
          </a>
          <a
            className="TopicSquare "
            href="/learn"
            onClick={[Function]}
          >
            <div
              className="TopicSquare__picture_container"
            >
              <div
                className="TopicSquare__picture TopicSquare__picture--locked"
                style={
                  Object {
                    "backgroundImage": "url(https://i.imgur.com/h6jaH4c.jpg)",
                  }
                }
              />
              <div
                className="TopicSquare__container TopicSquare__container--locked"
              >
                Coming soon
              </div>
            </div>
          </a>
          <a
            className="TopicSquare "
            href="/learn"
            onClick={[Function]}
          >
            <div
              className="TopicSquare__picture_container"
            >
              <div
                className="TopicSquare__picture TopicSquare__picture--locked"
                style={
                  Object {
                    "backgroundImage": "url(https://i.imgur.com/h6jaH4c.jpg)",
                  }
                }
              />
              <div
                className="TopicSquare__container TopicSquare__container--locked"
              >
                Coming soon
              </div>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
